### ISSUE TITLE
Read data from SSL-Refbox

### ISSUE THREAD

**Issue by MathewMacDougall** [UBC-Thunderbots/Software#72]
In addition to SSL vision data that gives us the state of the world (robot positions, etc.) we also need to listen for commands from the Referee Box.

The refbox from previous years can be found here: https://github.com/RoboCup-SSL/ssl-refbox
There had been talk about re-writing it for RoboCup 2019. The new repository can be found here: https://github.com/RoboCup-SSL/ssl-game-controller

There is sample client code in the RefBox repositories. Our code can likely be based very closely on that, and will likely be very similar to our code that receives SSL-Vision data (see the SSLVisionClient in the `backend_input` node)

Data that is received should be stored in the datastructure created in #43 , so this ticket is partially blocked by #43 . Nearly all the work can be done without the datastructure however.

**Comment by MathewMacDougall** [comment]
_We're T-7 days to the end of October, so just doing a quick run-through to see if there's anything we can afford to push to an November sprint._

Ideally we should have this done for October, since it's a pretty direct feature of this Milestone's goal of a connected system. @jordanzliu think you can squeeze this in to this month? It should be pretty straightforward as I expect it to mostly copy the new `SSL_Vision_Client` from #85

**Comment by MathewMacDougall** [comment]
One thing I noticed in RoboTeam Twente's 2017 TDP was that they apparently have a "refbox filter", at least according to a high-level diagram. Might be worth looking into exactly what this is and if it's useful/necessary (would likely be a separate ticket if so).
